modules/phase_field/doc/content/modules/phase_field/Nucleation/DiscreteNucleation.md

```diff
@@ -27,16 +27,78 @@ adaptivity, and a postprocessor:
+- [DiscreteNucleationForce](/DiscreteNucleationForce.md) - a kernel to map the
+  [`DiscreteNucleationMap`](/DiscreteNucleationMap.md) to a variable field in
+  combination with a [Reaction](/Reaction.md) kernel.
 
-## Discussion
+## Nucleation approaches
 
-The nucleation free energy penalty is added to the physical free energy
-contributions of the system using a
-[`DerivativeSumMaterial`](/DerivativeSumMaterial.md)
+All approaches require the use of an inserter, and a map user object. Advanced
+users may chose to derive their own userobjects from the provided classes to,
+for example, implement maps that realize different nucleus shapes than the
+smooth interface spheres provided by
+[`DiscreteNucleationMap`](/DiscreteNucleationMap.md).
 
-## Example
+Multiple options exist for applyting the nucleus data to the variable fields in
+the simulation.
 
-An example input file is located at
+### Free energy penalty based nucleation
 
-!listing modules/phase_field/examples/nucleation/refine.i
+The free energy penalty based approaces eschew directly modifying conserved
+concentration and non-conserved order parameter fields. Instead they bias the
+thermodynamocs of the system in a way that drives the formation of nuclei at the
+locations provided by the discrete nucleation system. Simply stated the local
+free energy density is modified to make the nucleated state a lower energy
+state. This causes solute to diffuse toward the nucleation site (in the
+conserved case) brings about a change in a non-conserved order parameter field.
+Moose provides the [`DiscreteNucleation`](materials/DiscreteNucleation.md)
+material, which implements a simple harmonic form of such a penalty, but
+advanced users are free to implement their own free energy modifications
+following the example set in this material.
+
+#### Conserved order parameters
+
+Use a [`DerivativeSumMaterial`](/DerivativeSumMaterial.md) to add the nucleation
+free energy penalty to the physical free energy contributions of the system
+(which is utilized by a Cahn-Hilliard kernel).
+
+#### Non-Conserved order parameters
+
+Use either a [`DerivativeSumMaterial`](/DerivativeSumMaterial.md) to add the
+nucleation free energy penalty to the physical free energy contributions of the
+system, or add an additional [`AllenCahn`](/AllenCahn.md) kernel using only the
+nucleation penalty energy to the variable influenced by the nucleation process.
+The latter approach also works for hardcoded free energies that do not utilize
+the [Function Material](/phase_field/FunctionMaterials.md) approach.
+
+### Direct order parameter modification
+
+In some cases, such as polycrystalline models used in conjunction with grain
+tracker, direct modification of non-conserved order parameters can be a feasible
+approach.
+
+Nucleation of new grains in a polycrystalline model with
+[GrainTracker](/GrainTracker.md) requires the use of reserved (or staging) order
```

Comment:
The concept of reserved/staging order parameters is not explained clearly here, it would be nice to add a sentence or two describing why you need them.  (Flesh out what is written here.)

Reply:
I added a comment but this really belongs on the linked grain tracker documentation page.